docs/management_api.yml

```diff
@@ -120,22 +154,59 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
-  /devices/{id}/check-update:
+  /devices/{id}/download:
     post:
       tags:
         - ManagementAPI
-      operationId: Check Update
-      summary: Trigger check-update for the Mender client running on the device
+      operationId: Download
+      summary: Download a file from the device
       parameters:
         - in: path
           name: id
           required: true
           schema:
             type: string
           description: ID for the target device.
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              type: object
+              properties:
+                path:
+                  type: string
+                  description: Path of the file on the target device.          
```

Comment:
technically speaking true, but it is "a path to the file to download", feel free to disregard if you do not like it.